{{cookiecutter.project_shortname}}/invenio.cfg

```diff
@@ -62,3 +62,11 @@ BABEL_DEFAULT_TIMEZONE = 'Europe/Zurich'
 THEME_FRONTPAGE_TITLE = "{{ cookiecutter.project_name }}"
 # Header logo
 THEME_LOGO="images/logo.svg"
+
+
+# Invenio-APP-RDM
+# ===============
+# See https://invenio-app-rdm.readthedocs.io/en/latest/configuration.html
+
+# Instance's theme entrypoint file. Path under `theme` folder.
+INSTANCE_THEME_FILE = './scss/{{ cookiecutter.project_shortname }}/theme.scss'
```

Comment:
I think what is throwing me off (and what might throw other developer off) is the `./` prefix: given the location of `invenio.cfg` I think it means there is a `scss/` folder next to `invenio.cfg` which there obviously isn't. Then, seeing where `INSTANCE_THEME_FILE` is used, I would think that's where the `scss` folder is (in  `invenio_app_rdm/theme`), but again it's not there. it is in `invenio_app_rdm/theme/assets/` ...

Reply:
@fenekku The variable there is pointing relatively to the `assets` folder. I saw that I have outdated information in the comment above. Was this the confusing part? Otherwise the convention `./scss` is used along the invenio modules I think...

Reply:
Updating the comment will help :+1: A new developer coming in won't know the `./` convention (I didn't!) and it's misleading (`./` implies reference to current directory but it's not), so my vote would be in removing it :).

Reply:
But [here](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/webpack.py#L19) is the same thing. It is implied that starts from the `assets` folder.

Reply:
yup and it is misleading there too ;) 

Let's have other people lean in on this too perhaps...

Reply:
@fenekku this is also used here https://github.com/inveniosoftware/invenio-theme/blob/master/invenio_theme/webpack.py#L32. The current directory is needed for webpack to resolve the entrypoints when is trying to bundle all assets. Therefore starts from the  root directory ( `/opt/var/instance/assets`) and resolves accordingly `scss` and `js` entry files. I understand the confusion and this is why I also tried to hide it in my first implementation. But I understood that your point was that you were more in favour of consistency instead of simplicity so I made it consistent :) But now I feel that you like more simplicity :P But I agree probably @ppanero and @lnielsen  can express their perspective too :)

Reply:
Hy there,

So my take on this is:

1- I am with Guillaume on the fact that this is confusing. However, it is as it has been done in the whole Invenio. Even though I would like the learning curve to be smoother, some things just need to be learnt. I think this is a "documentation" issue. We should document it in some sort of "Climber docs" (analogy to a dev climbing the steep learning curve ;) )

2- I would prefer simplicity over consistency. I would go with simplicity in RDM. However, refactoring everything else in Invenio is not doable due to man power and time constraints (We still need to have in mind the deadlines).

Conclusion: Since now it is consistent I would merge it as is and open an issue for when we work further on the dev experience. Mainly cuz it is blocking progress and it is not bad, just Invenio things that need to be re-evaluated (e.g. through and RFC).

WDYT guys?

Reply:
Works for me. Let's get this unblocked by adding a comment about it and come back to the path later if others are puzzled like me (I could genuinely be the only one, in which case, we don't even have to come back to it :) ).